src/OpenTracing/Tracer.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace OpenTracing;
+
+use OpenTracing\Exception\InvalidFormatException;
+
+/**
+ * Tracer is the entry point API between instrumentation code and the
+ * tracing implementation.
+ *
+ * This implementation both defines the public Tracer API, and provides
+ * a default no-op behavior.
+ *
+ * @package OpenTracing
+ */
+abstract class Tracer
+{
+    /**
+     * @var Tracer
+     */
+    private static $globalTracerInstance = null;
+
+    /**
+     * Starts and returns a new Span representing a unit of work.
+     *
+     * @param string $operationName
+     * @param Span $parent
+     * @param array $tags
+     * @param float $startTime
+     * @return Span
+     */
+    abstract public function startSpan($operationName = null, $parent = null, $tags = null, $startTime = null);
+
+    /**
+     * Returns an Injector instance corresponding to $format
+     *
+     * @throws InvalidFormatException
+     *
+     * @param string $format
+     * @return Injector
+     */
+    abstract public function injector($format);
```

Comment:
FYI, the spec has been simplified to remove the Injector/Extractor concept and move `Inject`/`Join` into `Tracer` itself... https://github.com/opentracing/opentracing.github.io/issues/70

Here are the diffs in [Go](https://github.com/opentracing/opentracing-go/pull/65) and [Python](https://github.com/opentracing/opentracing-python/pull/20).